### DISCUSSION TITLE
Getting valueFrom.secretKeyRef.name invalid value

### DISCUSSION THREAD

**Discussion by NChittimalla** [airbytehq/airbyte#45453] [Deployment]
helm install airbyte airbyte/airbyte --namespace airbyte --values ./values.yaml --timeout 60m
Error: INSTALLATION FAILED: 3 errors occurred:
        * Deployment.apps "airbyte-server" is invalid: [spec.template.spec.containers[0].env[40].valueFrom.secretKeyRef.name: Invalid value: "": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*'), spec.template.spec.containers[0].env[41].valueFrom.secretKeyRef.name: Invalid value: "": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')]
        * Deployment.apps "airbyte-worker" is invalid: [spec.template.spec.containers[0].env[47].valueFrom.secretKeyRef.name: Invalid value: "": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*'), spec.template.spec.containers[0].env[48].valueFrom.secretKeyRef.name: Invalid value: "": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')]
        * Deployment.apps "airbyte-workload-launcher" is invalid: [spec.template.spec.containers[0].env[62].valueFrom.secretKeyRef.name: Invalid value: "": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*'), spec.template.spec.containers[0].env[63].valueFrom.secretKeyRef.name: Invalid value: "": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')]
        
 Anyone please help, iam getting this error when i try to deploy airbyte application in EKS, i tried many possibilities but cant overcome it.
 
 Please find the attachment of my values.yaml file.
 
[values.yaml.pdf](https://github.com/user-attachments/files/16992752/values.yaml.pdf)

**Comment by marcosmarxm** [comment]
Please don't duplicate posts.
Close as duplicated of https://github.com/airbytehq/airbyte/discussions/45386
